Assets/Scripts/Satellit.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Satellit : MonoBehaviour
+{
+    public bool isLiving;
+    public GameObject Effect;
```

Comment:
```suggestion
    public GameObject effect;
```
Vermeide Variable Namen die mit einem Großbuchstaben anfangen, siehe [Coding Convention von C#](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions)